design/components/card.stories.tsx

```diff
@@ -0,0 +1,40 @@
+import { Story } from '@storybook/react/types-6-0';
+import React from 'react';
+
+import duurzaamheid from './assets/duurzaamheid.png';
+import { CardProps, Card } from './card';
+import { RichText } from './rich-text';
+import { Tag } from './tag';
+import { Title } from './title';
+
+export default {
+  title: 'Example/Card',
+  component: Card,
+};
+
+const Template: Story<CardProps> = (args): React.ReactElement => (
+  <Card {...args} />
+);
+
+export const cardWithImageAndText = Template.bind({});
+cardWithImageAndText.args = {
```

Comment:
Could you stick to TitleCase for the story names.

`CardWithImageAndText `

Reply:
you said no case  or story ?maybe i'm missing the point 🤔

Reply:
done